widget/src/components/Auth/LoginButton.tsx

```diff
@@ -66,9 +66,23 @@ class LoginButton extends React.Component<Props, State> {
     if (this.state.isRedirectingToAuthorizeUrl) return;
     this.setState({isRedirectingToAuthorizeUrl: true});
 
-    window.location.href = loginAPI.createLoginURL(AuthorizationCodeExchangeStateHelper.toJSONString({
-      path: appHistory.location.pathname,
-    }));
+    window.location.href = loginAPI.createLoginWebsiteUrl({
```

Comment:
might consider using `location.replace` instead of this `location.href` reassignment since this is a redirection